Spravo/Services/ContactsProvider.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  AddressBookViewModel.swift
+//  Spravo
+//
+//  Created by Onix on 10/4/19.
+//  Copyright © 2019 Home. All rights reserved.
+//
+
+import Foundation
+
+protocol ContactsProviderType: Service {
+    var userModel: User { get }
+    var contactsModel: [Contact] { get }
```

Comment:
It would be right to say `contactModels`. Ultimately, it would be right to say, just, `contacts`